hypothesisfuzzer/fuzzer.py

```diff
@@ -8,67 +8,71 @@
 from flask_sqlalchemy import SQLAlchemy
 
 
-app = Flask(__name__)
-app.config['SQLALCHEMY_DATABASE_URI'] = \
-    os.environ.get('DATABASE_URL', 'sqlite:///data.db')
-app.config['SQLALCHEMY_TRACK_MODIFICATION'] = False
-db = SQLAlchemy(app)
-
-current_fuzzing_task = None
+class Fuzzer:
 
+    def __init__(self):
+        self.app = Flask(__name__)
+        self.app.config['SQLALCHEMY_DATABASE_URI'] = \
+            os.environ.get('DATABASE_URL', 'sqlite:///data.db')
+        self.app.config['SQLALCHEMY_TRACK_MODIFICATION'] = False
+        self.db = SQLAlchemy(self.app)
+        self.current_fuzzing_task = None
 
-@app.route('/webhook', methods=['POST'])
-def on_git_push():
+    def run(self, **kwargs):
+        self.app.run(**kwargs)
+        self.db.create_all()
 
-    global current_fuzzing_task
-    if current_fuzzing_task:
-        current_fuzzing_task.running = False
-        current_fuzzing_task.join()
+        @self.app.route('/webhook', methods=['POST'])
+        def on_git_push():
 
-    if os.path.exists("code"):
-        shutil.rmtree("code", ignore_errors=True)
+            global current_fuzzing_task
 
-    if os.path.isfile("results.txt"):
-        os.remove("results.txt")
+            if current_fuzzing_task:
```

Comment:
Shouldn't this be self.current_fuzzing_task. Where do you assign it? It would be None forever?